src/Skoruba.IdentityServer4.Admin.EntityFramework.Identity/Repositories/Interfaces/IIdentityRepository.cs

```diff
@@ -52,6 +52,8 @@ public interface IIdentityRepository<TUserKey, TRoleKey, TUser, TRole, TKey, TUs
 
         Task<PagedList<TUserClaim>> GetUserClaimsAsync(string userId, int page = 1, int pageSize = 10);
 
+        Task<TUserClaim> GetUserClaimByTypeAsync(string userId, string claimType);
```

Comment:
Adding the contract for the ability to search for a user claim bin "type" text instead of by id (again, the example is that the "invitation_token" claim type needs to be found before they are allowed to "RegisterByInvitation".